src/cmd-buildextend-metal

```diff
@@ -128,7 +128,11 @@ size="$(jq '."estimate-mb".final' "$PWD/tmp/ostree-size.json")"
 size="$(( size + 513 ))M"
 echo "Disk size estimated to $size"
 kargs="$(python3 -c 'import sys, yaml; args = yaml.safe_load(sys.stdin).get("extra-kargs", []); print(" ".join(args))' < "$configdir/image.yaml")"
-kargs="$kargs console=tty0 console=${VM_TERMINAL},115200n8 ignition.platform.id=metal"
+tty="console=tty0 console=${VM_TERMINAL},115200n8"
+if [ "$arch" == "s390x" ]; then
+    tty="console=${VM_TERMINAL}"
```

Comment:
so the moral of the story is that s390x doesn't support `tty0` and also doesn't like the `115200n8` baude specification? Can we get a comment in the code that to that effect?

Reply:
This one is basically the same as : `https://github.com/coreos/coreos-assembler/blob/master/src/cmd-build#L278`.
Just wanna iterate myself in the previous PR, s390x doesn't even require `console=` parameter. On different hypervisors, the tty is automatically detected, and _to my knowledge_, there is only one tty for each hypervisor, so no need to tell. Current `VM_TERMINAL` is `ttysclp0` which is KVM path, which is ok for now. We need to do more testing and probably just drop `console=` for s390x in near future.